docs/user/filesystem.md

```diff
@@ -376,6 +376,32 @@ These are two command line options to control the DFuse process itself.
 These will affect all containers accessed via DFuse, regardless of any
 container attributes.
 
+### Permissions
+
+DFuse can serve data from any user's container, but needs appropriate permissions in order to do
+this.
+
+File ownership within containers is set by the container being served, with the owner of the
+container owning all files within that container, so if looking at the container of another user
+then all entries within that container will be owned by that user, and file-based permissions
+checks by the kernel will be made on that basis.
+
+Should write permission be granted to another user then any newly created files will also be
+owned by the container owner, regardless of the user used to create them.  Permissions are only
+checked on connect, so if permissions are revoked users need to
+restart DFuse for these to be picked up.
+
+#### Pool permissions.
+
+DFuse needs 'r' permission for pools only.
```

Comment:
I am not sure I get this... Is it a real limitation or do we enforce it ??

Reply:
It will fail if you don't have r permission.

Reply:
This might be better working?
```suggestion
DFuse only needs 'r' permission for pools.
```